src/components/Book.js

```diff
@@ -0,0 +1,37 @@
+import { CircularProgressbar } from 'react-circular-progressbar';
+import classes from './Books.module.css';
+import 'react-circular-progressbar/dist/styles.css';
+
+const percentage = 66;
+const Book = () => (
+  <>
+    <div className={classes.bookCard}>
+      <div className={classes.bookInfo}>
+        <div>
+          <p className={classes.catagory}>The Hunger Games</p>
+          <h3 className={classes.title}>Suzanne Collins</h3>
+          <p className={classes.author}>Action</p>
+        </div>
+        <ul className={classes.action}>
+          <li>Comment</li>
+          <li>Remove</li>
+          <li>Edit</li>
+        </ul>
```

Comment:
- [ ] This component should have a `button` for removing books currently you are using an `li` which isn't semantically correct. please make this adjustment.